tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/RcmBase.cs

```diff
@@ -14,13 +16,17 @@ public class RcmBase : AspNetBase
     protected const string LogFileNamePrefix = "dotnet-tracer-managed-";
 
 #pragma warning disable SA1401
+    protected readonly string logDirectory;
     protected readonly TimeSpan logEntryWatcherTimeout = TimeSpan.FromSeconds(20);
 #pragma warning restore SA1401
 
     public RcmBase(ITestOutputHelper outputHelper, string testName)
         : base("AspNetCore5", outputHelper, "/shutdown", testName: testName)
     {
         SetEnvironmentVariable(ConfigurationKeys.Rcm.PollInterval, "500");
+        var logPath = DatadogLogging.GetLogDirectory();
+        logDirectory = Path.Combine(logPath, $"{testName}Logs");
```

Comment:
Since back-to-back runs will write logs into the same directory, do we need to clear out the directory before starting the test? Or is that unnecessary because only updates to log files will be detected?

Reply:
we dont need to clear it out as the log entry watcher will read it to the end first in its constructor, if it finds the file